api/v1alpha1/conversion_test.go

```diff
@@ -145,7 +145,6 @@ func overrideVirtualMachineClassFieldsFuncs(codecs runtimeserializer.CodecFactor
 			// TODO: Need to save serialized object to support lossless conversions.
 			classStatus.Capabilities = nil
```

Comment:
While here, can we just enable conversion of these fields too? Sure, TBD if we'll end to supporting these in time but doing it now should be pretty low cost w.r.t. the rest of the change.

Reply:
Done